tests/integration/data/polarion/existing_testcase/main.fmf

```diff
@@ -3,4 +3,5 @@ extra-nitrate: TC#0609686
 link:
   - verifies: https://bugzilla.redhat.com/show_bug.cgi?id=1925518
   - implements: https://polarion.example/polarion/#/project/TMT/workitem?id=TMT-43
+  - implements: https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL-293393
```

Comment:
either make it .example like ^ or IMO you can remove this line completely so we are not putting internal links on public github

right now it doesn't use the links for anything anyway